### PR TITLE
Fix available funds

### PR DESCRIPTION
**Pull request by willm30**
## Description

This PR fixes the disappearance of "Available Funds" in ColonyHome.

[A change to the colony fragment](https://github.com/JoinColony/colonyCDapp/commit/a2087b97fc203e75a7f8c6891c6eb7dc41edff10) made the token we're adding manually in the cache the wrong format, and so apparently invalidated our manual changes to the Colony field in the cache.

**Changes** 🏗

* Update token format in cache and add typing so we notice more quickly and easily in the future. 

## Testing

On master, you should see your available funds.

Once I get some approvals I will push the changes directly to qa also so no need to rebase to see the fix.

### CODE REVIEW

File: src/graphql/fragments/colony.graphql [hunk 1]
```diff
@@ -47,6 +42,15 @@ fragment Colony on Colony {
   }
 }
 
+fragment ColonyTokensConnection on ModelColonyTokensConnection {
```
Comment (willm30) [inline src/graphql/fragments/colony.graphql:45]: Making this a fragment lets it be used as a type which will make it obvious if something like this happens again in the future

File: src/cache/colony/tokens.ts [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
   tokens: {
-    read: (baseTokens) => ({
-      ...baseTokens,
-      items: [
-        ...baseTokens.items,
+    read: (baseTokens) => {
+      const updatedTokens: UnaliasedColonyTokensItem[] = [
         {
           __typename: 'ColonyTokens',
+          id: 'DEFAULT_TOKEN_ID',
```
Comment (willm30) [inline src/cache/colony/tokens.ts:14]: This was the offender!

**Review by jakubcolony — APPROVED** [review]
nice one, thanks for fixing this!

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👌

**Pull request merged**